modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-about_{context}"]
+= About performing a canary rollout update
+
+This topic describes the general workflow of this canary rollout update process. The steps to perform each task in the workflow are described in the following sections.
+
+. Create MCPs based on the worker pool. The number of nodes in each MCP depends on a few factors, such as your maintenance window duration for each MCP, and the amount of reserve capacity (extra worker nodes) available in your cluster.
++
+[NOTE]
+====
+You can change the `maxUnavailable` setting in an MCP to specify the percentage or the number of machines that can be updating at any given time. The default is 1.
+====
+
+. Add a node selector to the custom MCPs. For each node that you do not want to update simultaneously with the rest of the cluster, add a matching label to the nodes. This label associates the node to the MCP.
++
+[NOTE]
+====
+Do not remove the default worker label from the nodes. The nodes *must* have a role label to function properly in the cluster.
+====
+
+. Pause the MCPs you do not want to update as part of the update process.
++
+[NOTE]
+====
+Pausing the MCP also pauses the kube-apiserver-to-kubelet-signer automatic CA certificates rotation. New CA certificates are generated at 292 days from the installation date and old certificates are removed 365 days from the installation date. See the link:https://access.redhat.com/articles/5651701[Understand CA cert auto renewal in Red Hat OpenShift 4] to find out how much time you have before the next automatic CA certificate rotation. Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the cert rotation does not happen, which causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`.
+====
+
+. Perform the cluster update. The update process updates the MCPs that are not paused, including the control plane nodes (also known as the master nodes).
```

Comment:
Per the IBM style guide, I would avoid using parentheses in general text. 

> Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text.

Reply:
This one is our style for transitioning from _master_ to _control plane_.

Reply:
@mburke5678 oh that's right! :) Thanks for the reminder.